app/Spago.hs

```diff
@@ -31,11 +30,20 @@ main = withUtf8 $ do
   Env.setEnv "GIT_TERMINAL_PROMPT" "0"
 
   (command, globalOptions@GlobalOptions{..}) 
-    <- CLI.optionsExt "" "" "Spago - manage your PureScript projects" (Text.pack spagoVersion) CLI.parser
+    <- CLI.options "Spago - manage your PureScript projects" CLI.parser
+
+  let handleDefault :: MonadIO m => ShowVersion -> m ()
+      handleDefault = \case
+        DoShowVersion -> CLI.echo spagoVersion *> exitSuccess
+        NoShowVersion -> pure ()
 
   Run.withEnv globalOptions $
     case command of
 
+      -- ### Commands that need no env
+      Default shouldShowVersion
+        -> handleDefault shouldShowVersion
```

Comment:
```suggestion
      Default showVersion -> handleDefault showVersion
```

Reply:
`showVersion` clashes with the [one from `Data.Version`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Data-Version.html#v:showVersion). I was between this choice and making `Data.Version` a qualified import. If it looks better to you, we can go with the latter :+1:

Reply:
Oh I see, this is fine then :+1: